source/development/packaging/generate-wazuh-kibana-app.rst

```diff
@@ -32,7 +32,7 @@ Execute the ``generate_wazuh_app.sh`` script, with the different options you des
 
   Usage: ./generate_wazuh_app.sh [OPTIONS]
 
-      -b, --branch <branch>     [Required] Select Git branch or tag e.g.v3.11.4-7.4.0
+      -b, --branch <branch>     [Required] Select Git branch or tag e.g.v|WAZUH_LATEST|-|ELASTICSEARCH_LATEST_OVA|
```

Comment:
Why OVA? It must be Elasticsearch latest.

Reply:
Same in all this document

Reply:
Done.